terraform/dev.tfvars

```diff
@@ -1,8 +1,6 @@
 aws_account = "dev"
-system_dns_zone_id = "Z1QGLFML8EG6G7"
-system_dns_zone_name = "dev.cloudpipeline.digital"
+env_dns_zone_id = "Z1QGLFML8EG6G7"
```

Comment:
We have removed system dns zone completely and introduced new variable that consists of old  var.env + var.system_dns_zone_name. Example: https://github.com/alphagov/paas-cf/pull/159/commits/b4078cc63d130bf1c8d6b3ed41f429a5b0935d2f#diff-88a715cd562c7412a9c5bf1909c04b56R22. To be consistent we have renamed terraform variables to match Makefile and Concourse variables. When it comes to naming, as you know it's hard :) . I picked this one as it refers to entire environment not just root or system domain.